storyEmirhan.md

```diff
@@ -4,4 +4,6 @@
 &nbsp;
 Rood kapje ging naar huis en zag dat haar oma was opgegeten door de grote boze wolf. 
 
+![Roodkapje](https://images.nypl.org/index.php?id=1704116&t=w)
```

Comment:
de foto is echt te lelijk

Reply:
ik heb de foto veranderd, kun je hem herzien